src/vm/amd64/externalmethodfixupthunk.S

```diff
@@ -11,6 +11,7 @@
 
 NESTED_ENTRY ExternalMethodFixupStub, _TEXT, NoHandler
 
+        set_cfa_register rsp, 8
```

Comment:
Oops, sure it would. It is probably too late for my brain to work properly today :-)